onboarding/maintainers.md

```diff
@@ -53,11 +53,13 @@ Being that contributions will vary in substance and frequency, the Core team wil
         - Feel free to fix small typos and wording using the in-browser editor
         -  https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews
     - New lessons will have to have the "seeds" file updated to accomodate and show on the site
-- **Updates** to the markdown will be reflected on the site **once a day**
-    - If somebody asks why their PR isn't showing, this is why
 - **Merge PRs**
+    - All maintainers share the responsibility of reviewing and merging PRs, even if another maintainer requested changes or created the issue (with the exception of a specialized epic project that one maintainer is overseeing, like the new HTML/CSS lessons).
```

Comment:
I think we should adjust the wording to this to give the meaning of: "If another maintainer has requested changes, and you are 100% confident those changes are fixed, feel free to merge. Otherwise ask.

Reply:
@I3uckwheat I understand your concern here, but I wonder if it is addressed in lines 62-64. Every wording that I am coming up with seems to rehash the points that I made there. I think the same goes for any PR that one does not understand.

Reply:
Ah, I see it now, my bad!